proposed/http-message.md

```diff
@@ -542,7 +558,7 @@ interface RequestInterface extends MessageInterface
      * @see MessageInterface::getHeader()
      * @see UriInterface::getHost()
      * @param string $name Case-insensitive header field name.
-     * @return string
+     * @return string[]
```

Comment:
Can you add a description of the return type? e.g., "Returns an array of strings"